launch/model_endpoint.py

```diff
@@ -148,9 +171,14 @@ def single_request(request):
 
     def status(self):
         """Gets the status of the ModelEndpoint.
-        TODO this functionality currently does not exist on the server.
+        
```

Comment:
copy pasta for now but i'd leave this split in case we have different fields in sync vs async